### PR TITLE
Allow SNES to look at notches (round 2)

### PR DESCRIPTION
**Pull request by zrolfs**

### CODE REVIEW

**Comment by zrolfs** [comment]
Hmm. The search took 60% longer than before, but got 3% more PSMs, so it's now better than the classic?? I'm pretty confused here.

**Comment by zrolfs** [comment]
Oh, the indexes being in a new folder made it remake the indexes. So it's not actually 60% slower! (But there are still more PSMs!).

**Comment by acesnik** [comment]
Can N64 look at notches, yet?

**Comment by zrolfs** [comment]
It was an intentional choice :)

**Comment by zrolfs** [comment]
This is ready!

File: EngineLayer/ModernSearch/ModernSearchEngine.cs [hunk 1]
```diff
@@ -70,7 +70,7 @@ protected override MetaMorpheusEngineResults RunSpecific()
                     // note that this is the OPPOSITE of the classic search (which calculates experimental masses from theoretical values)
                     // this is just PRELIMINARY precursor-mass filtering
                     // additional checks are made later to ensure that the theoretical precursor mass is acceptable
-                    var notches = MassDiffAcceptor.GetAllowedPrecursorMassIntervals(scan.PrecursorMass);
+                    var notches = MassDiffAcceptor.GetAllowedPrecursorMassIntervalsFromTheoreticalMass(scan.PrecursorMass);
```
Comment (rmillikin) [inline EngineLayer/ModernSearch/ModernSearchEngine.cs]: does this need to be `GetAllowedPrecursorMassIntervalsFromObservedMass`?

Comment (zrolfs) [reply EngineLayer/ModernSearch/ModernSearchEngine.cs]: Yes, it should be. Good catch.
Curious that this wasn't caught in the modern/classic comparison before. Perhaps no missed mono in that test?

Comment (zrolfs) [reply EngineLayer/ModernSearch/ModernSearchEngine.cs]: Ahh, The "Observed" code was just happening below it. I've simplified it now.

**Pull request merged**